### DISCUSSION TITLE
How can I repeat an animation iteration once?

### DISCUSSION THREAD

**Discussion by gyull0210** [tailwindlabs/tailwindcss#8925] [Help]
When tailwindcss custom animation is specified, it is not applied without infinite. What should I do?
How does tailwindcss specify the animation-iteration-count property that exists in the CSS?

**Comment by wongjn** [comment]
As per the [documentation](https://tailwindcss.com/docs/animation), the `animation` properties are all specified in the animation utility value using the [`animation` shorthand CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/animation). If you want to customize the `animation-iteration-count`, you can set it in the value.

## Examples

### Re-use animation that exists in Tailwind core

```js
module.exports = {
  theme: {
    extend: {
      animation: {
        'spin-twice': 'spin 3s linear 2',
      },
    },
  },
};
```
We re-use the `spin` keyframe, but set the `animation-iteration-count` to `2`.

### Custom animation
```js
module.exports = {
  theme: {
    extend: {
      animation: {
        foo: 'foo 1s 42',
      },
      keyframes: {
        foo: {
          '0%': {
            // …
          },
          '100%': {
            // …
          },
        },
      },
    },
  },
};
```
We make our own animation along with setting the `animation-iteration-count` to `42`.
